recipes/openssh/files/init

```diff
@@ -1,6 +1,8 @@
 #! /bin/sh
 set -e
 
+HOST_KEY_TYPES="rsa dsa"
```

Comment:
Well, the idea is that I don't want to maintain the list in two places, so I decided to leave something different, while still innnocent looking in this location, as it gets rewritten by the USE flag anyway.